pkgs/top-level/all-packages.nix

```diff
@@ -4910,6 +4910,8 @@ in
 
   librest = callPackage ../development/libraries/librest { };
 
+  libslirp = callPackage ../tools/networking/libslirp { };
```

Comment:
This is not a tool (it provides no binaries): please move its files to ../development/libraries (and its definition near libsigsegv).